openmetadata-service/src/main/java/org/openmetadata/service/jdbi3/CollectionDAO.java

```diff
@@ -3442,4 +3446,166 @@ default String getNameColumn() {
       return "name";
     }
   }
+
+  interface WorkflowDAO extends EntityDAO<Workflow> {
+    @Override
+    default String getTableName() {
+      return "operations_workflow";
+    }
+
+    @Override
+    default Class<Workflow> getEntityClass() {
+      return Workflow.class;
+    }
+
+    @Override
+    default String getNameColumn() {
+      return "name";
+    }
+
+    @Override
+    default List<String> listBefore(ListFilter filter, int limit, String before) {
```

Comment:
We allow to list worflows by type or status. Keeping things ready for when we add new operations in the backend